src/components/CardCVCContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import Input from './common/input/Input';
+import { ErrorWrapper, ErrorText } from '../styles/common';
+import InputField from './common/inputField/InputField';
+import { ErrorDetail } from '../types/error';
+
+interface CardCVCContainerProps {
+  value: string;
+  handleChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  handleBlur: () => void;
+  errorInfo: ErrorDetail;
+  setIsCardFront: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const CardCVCContainer = ({
+  value,
+  handleChange,
+  handleBlur,
+  errorInfo,
+  setIsCardFront,
+}: CardCVCContainerProps) => {
+  return (
+    <div>
+      <InputField title="CVC 번호를 입력해 주세요" labelText="CVC" labelFor="cvc-input">
+        <Input
+          id="cvc-input"
+          isError={errorInfo.isError}
+          value={value}
+          onChange={handleChange}
+          onBlur={handleBlur}
+          onFocus={() => setIsCardFront(false)}
+          placeholder="123"
+          width="100%"
+          maxLength={4}
+          autoFocus={true}
```

Comment:
간결하게 이렇게만 써도 좋을 것 같아용 :)

```suggestion
          autoFocus
```

Reply:
> chore(CardCVCContainer): cvc autoFocus 추가

사소한데, 프로덕션 기능에 변경사항이 있으니, `chore`보다 `feat` 커밋으로 작성해주시는 게 더 적절할 것 같습니다 :)

Reply:
오 수정하겠습니다!!